src/Logo - Azure Dev Ops - Color.svg

```diff
@@ -0,0 +1,12 @@
+<svg width="24" height="24" viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg">
+<path d="M20 6.1875V16.5363L15.75 20.0212L9.1625 17.62V20L5.43313 15.1231L16.3025 15.9731V6.655L20 6.1875ZM16.3769 6.70812L10.2781 3V5.43313L4.67875 7.08L3 9.23688V14.135L5.40125 15.1975V8.91813L16.3769 6.70812Z" fill="url(#paint0_linear)"/>
+<defs>
+<linearGradient id="paint0_linear" x1="11.5" y1="19.9681" x2="11.5" y2="3.03188" gradientUnits="userSpaceOnUse">
```

Comment:
Note this has the same problem as #22

Reply:
Yup, and I'm not sure if there's a way to get around it, because you can't define a gradient in a `fill="..."` attribute.

Reply:
The only way I can think of is to apply the gradient via a CSS `linear-gradient()`. But that would require modifying the definition of the logo, which is a no-no.

Reply:
I think we'll run into IDs for any icon with SVG-defined gradients (which will most likely be logos).

Reply:
If we move to loading the logos as `img`, that should solve that problem. Our regular icons shouldn't have gradients (and if we ever decide to change that we'll need to come up with a strategy for the gradient IDs)